internal/version/version.go

```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	// Version shows the current notation version, optionally with pre-release.
-	Version = "v0.7.1-alpha.1"
+	Version = "v1.0.0-alpha.3"
```

Comment:
We should probably not bump major version to v1.0.0 right now, the alpha releases will have patch updates. Which will bump 1.0.0-{prereleasetag} to 1.0.x-{prereleasetag} and require the v1 release to use the same patch version at some point, this would be bit confusing (e.e. v1.0.5-alpha.3 vs 1.0.5)

https://stackoverflow.com/questions/51373151/releasing-alpha-to-beta-to-production-with-semver

```suggestion
	Version = "v0.8.0-alpha.3"
```